config/utils.py

```diff
@@ -3,21 +3,5 @@
 
 from pynini import *
 
-DEFAULT_LATIN = union("a", "b", "c", "d", "e", "f", "g", "h", "i",
-                      "j", "k", "l", "m", "n", "o", "p", "q", "r",
-                      "s", "t", "u", "v", "w", "x", "y", "z")
-
 DEFAULT_INITIAL_PUNCTUATION = union("\"", "'")
 DEFAULT_FINAL_PUNCTUATION = union("!", "\"", ",", ".", ":", ";", "?")
-GEEZ_FINAL_PUNCTUATION = union("፠", "፡", "።", "፣", "፤",
-                               "፥", "፦", "፧", "፨")
-
-WESTERN_ARABIC_NUMERALS = union("0", "1", "3", "4", "5", "6", "7", "8", "9")
-GEEZ_NUMERALS = union("፩", "፪", "፫", "፬", "፭",
```

Comment:
Did you mean to delete the GEEZ_NUMERALS?

Reply:
And GEEZ Punctuation?

Reply:
Yep! Those got moved to the Amharic config. Though now that I think about it maybe they _should_ stay in `utils.py` in teh event that we want to normalize a different language that uses Ge'ez. Shall I move them back?